### PR TITLE
[redesign] Set page titles through router

### LINKED ISSUE #1412: new design lacking <title>
The new design is missing <title> from its pages.

### PR DESCRIPTION
**Pull request by thi4go**
closes #1412

### CODE REVIEW

File: src/containers/Routes/RouteWithTitle.jsx [hunk 1]
```diff
@@ -0,0 +1,9 @@
+import React from "react";
+import { Route } from "react-router-dom";
+
+const RouteWithTitle = ({ title, ...props }) => {
```
Comment (fernandoabolafio) [inline src/containers/Routes/RouteWithTitle.jsx]: Let's just call it Route, we may want to add other functionalities to it in the future and it is a bit clear IMO.

File: src/containers/Routes/RouteWithTitle.jsx [hunk 2]
```diff
@@ -0,0 +1,9 @@
...
+  document.title = title || "Politeia";
```
Comment (fernandoabolafio) [inline src/containers/Routes/RouteWithTitle.jsx]: The default title needs to come from the config `const { title } = useConfig()`

**Comment by thi4go** [comment]
Thanks for the quick review, @fernandoabolafio! Addressed comments, and I'll wait up on #1437 and update this PR.

**Comment by fernandoabolafio** [comment]
@thi4go it just needs some cleanup and conflicts solving before we merge.

File: src/containers/Routes/Route.jsx [hunk 3]
```diff
@@ -0,0 +1,10 @@
+import React from "react";
+import { useDocumentTitle } from "src/hooks/utils/useDocumentTitle";
+import { Route as Router } from "react-router-dom";
```
[the remark this answers is not in the document]
Comment (thi4go) [reply src/containers/Routes/Route.jsx]: Agreed mate, updated it! 👍

**Comment by thi4go** [comment]
Good to go! @fernandoabolafio 👍

**Pull request merged**